erpnext/manufacturing/doctype/work_order/work_order.json

```diff
@@ -107,6 +107,8 @@
   },
   {
    "depends_on": "eval:doc.production_item",
+   "fetch_from": "production_item.item_name",
```

Comment:
This looks good. You need to change the `modified` key of JSON file so this change is migrated.

Reply:
Thank's, I try again and I can confirm that 2 changes are needed, job_card.json with the strange fetch from "work_order.production_item.item_name" (that should not work but it does..., may be as field production_item is type ReadOnly and not Link  like as in future version) populated by  and not link in this version) and and work_order.json. 

This is the type of bug fix in 13 (and develop) but not in 12, It's really strange for me that fix in develop first, but I'll manage to play as your rule